source/2019-04-05-the-ember-times-issue-92.md

```diff
@@ -43,10 +43,13 @@ Curious to learn more? Check out her post on Medium: [3 reasons why you should v
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [Dependency Lint RFC ✨](https://github.com/emberjs/rfcs/pull/464)
+
+[@Alonski](https://github.com/Alonski) proposed that the addon `ember-cli-dependency-lint` be added to the default blueprint.
+
+This lint helps us detect early if our addons rely on the same package but use different versions. In such cases, Ember CLI merges all versions into one. This can cause our app to behave unexpectedly and become hard to debug. 😨😰😱
+
+You can learn about `ember-cli-dependency-lint` addon and how to resolve dependency conflicts at [https://github.com/salsify/ember-cli-dependency-lint](https://github.com/salsify/ember-cli-dependency-lint).
```

Comment:
I think it would be really cool to mention the RFC a last time with a call-to-action linking to it. E.g.

```suggestion
You can learn about `ember-cli-dependency-lint` addon and how to resolve dependency conflicts at [https://github.com/salsify/ember-cli-dependency-lint](https://github.com/salsify/ember-cli-dependency-lint) and be sure to leave your comments on [the current proposal to include it by default](https://github.com/emberjs/rfcs/pull/464).
```

What do you think?

Reply:
@jessica-jordan I made changes based on your suggestions. Thanks again!